pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -816,16 +809,40 @@ func BenchmarkTestFitScore(b *testing.B) {
 			},
 		},
 		{
-			name: "1000nodes/MostAllocated",
+			name: "RequestedToCapacityRatio with 10 resources",
 			requestedPod: st.MakePod().
-				Req(map[v1.ResourceName]string{"cpu": "1000", "memory": "2000"}).
+				Req(map[v1.ResourceName]string{"cpu": "3000", "memory": "5000"}).
```

Comment:
Do we have any reason we have to use `{"cpu": "3000", "memory": "5000"}`, if not, change to `{"cpu": "1000", "memory": "2000"}` and make it a local variable not a field of struct.